packages/components/error-message/template.njk

```diff
@@ -1,6 +1,9 @@
+{% set visuallyHiddenText = params.visuallyHiddenText | default("Error") -%}
+
 <span class="nhsuk-error-message
 {%- if params.classes %} {{ params.classes }}{% endif %}"
 {%- if params.id %} id="{{ params.id }}"{% endif %}
 {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
+  {% if visuallyHiddenText %}<span class="nhsuk-u-visually-hidden">{{ visuallyHiddenText }}:</span> {% endif -%}
```

Comment:
@AdamChrimes - Do you think we should hardcode the ":" or allow/rely on the user to put any punctuation they want?

Reply:
@mcheung-nhs I have a feeling people will miss it out and will end up with no punctuation. Can you think of an alternative punctuation people will use than `:`?

Reply:
@AdamChrimes - fair enough. Just don't want people to put in any extra punctuation at the end of the text so having a colon there doesn't make sense, although this is probably highly unlikely!